aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -2291,6 +2314,20 @@ func resourceAviatrixTransitGatewayUpdate(d *schema.ResourceData, meta interface
 		}
 	}
 
+	if d.HasChange("enable_jumbo_frame") {
+		if d.Get("enable_jumbo_frame").(bool) {
+			err := client.EnableJumboFrame(gateway)
+			if err != nil {
+				return fmt.Errorf("could not enable jumbo frame for gateway: %v", err)
```

Comment:
Error messages need to be unique strings for debugging/diagnosing issues. You have used the same string `could not enable jumbo frame for gateway` across the 3 gateway resources, if a customer reported a problem and all we had was this error string, we would have no clue which gateway resource had the problem. Also you reused the same string for Create and Update, which is also ambiguous.

Add more context to make the strings unique, for example, this is Transit Gateway update so the error should be
```
could not enable jumbo frame for transit gateway when updating
```

Reply:
I added unique messages for the error messages so you can tell if it's a gateway, spoke gateway or transit gateway. Also differentiated create and update error messages.